docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -275,11 +275,7 @@ This demo replaces it with the [Pagination](/components/pagination/) component.
 
 ### Toolbar
 
-The grid exposes props to hide specific elements of the UI:
-
-- `hideToolbar`: If `true`, the toolbar component is hidden.
-
-You can replace the default toolbar by providing your own implementation.
+You can replace the default toolbar by providing a `header` component.
```

Comment:
I'm lost with this. I thought we would add a toolbar slot

Reply:
This PR is 3 months old